source/newcommitter.md

```diff
@@ -227,21 +236,40 @@ sent after a positive result from the vote for a new committer.
     Of course, you can decline and instead remain as a 
     contributor, participating as you do now.
 
-    A. This personal invitation is a chance for you to 
-    accept or decline in private.  Either way, please 
-    let us know in reply to the private@[PROJECT].apache.org 
-    address only.
+    This personal invitation is a chance for you to accept or decline in private.
+    Please let us know in reply to this message whether you accept or decline.
+    
+    A. If you accept, you will need an Apache account (id) with privileges.
+    Please follow these instructions.
+    
+        1. If you already have an Apache account, let us know your id and we
+    will grant you privileges on the project repositories.
+
+        2. If you have previously sent an iCLA, let us know the email address
```

Comment:
```suggestion
        2. If you have previously sent an ICLA, let us know the email address
```

Same with the rest of this email template.

Reply:
Either iCLA or ICLA is used and I did not want to overly complicate this PR.